packages/frontend-2/components/project/page/models/Card.vue

```diff
@@ -24,85 +41,53 @@
         />
         <template v-else-if="previewUrl">
-          <NuxtLink :href="finalModelUrl" class="w-full h-full">
+          <div
+            class="bg-foundation-page w-full h-48 rounded-xl border border-outline-2"
+          >
             <PreviewImage :preview-url="previewUrl" />
-          </NuxtLink>
+          </div>
         </template>
         <div
           v-if="!isPendingModelFragment(model)"
           v-show="!previewUrl && !pendingVersion"
-          class="h-full w-full p-4"
+          class="h-48 w-full"
         >
           <ProjectCardImportFileArea
             ref="importArea"
             :project-id="projectId"
             :model-name="model.name"
-            class="h-full w-full"
+            class="w-full h-full"
           />
         </div>
       </div>
-      <div
-        class="h-auto sm:h-12 flex flex-col sm:flex-row sm:items-center px-2 py-1 gap-x-1"
-      >
-        <NuxtLink class="min-w-0 max-w-full cursor-pointer" :href="finalModelUrl">
-          <div
-            v-if="nameParts[0]"
-            class="text-xs text-foreground-2 relative -mb-1 truncate"
-          >
-            {{ nameParts[0] }}
-          </div>
+      <div class="flex justify-between items-center w-full h-8 pl-2">
+        <ProjectPageModelsCardUpdatedTime
+          class="text-body-3xs text-foreground-2"
+          :updated-at="updatedAtFullDate"
+        />
+        <div class="flex items-center gap-1">
           <div
-            class="font-bold text-sm sm:text-base truncate text-foreground flex-shrink min-w-0"
+            v-if="!isPendingModelFragment(model)"
+            class="flex items-center gap-1 !text-foreground-2"
+            :to="modelVersionsRoute(projectId, model.id)"
           >
-            {{ nameParts[1] }}
+            <IconDiscussions class="h-4 w-4" />
+            <span class="text-body-2xs font-medium">
+              {{ model.commentThreadCount.totalCount }}
+            </span>
           </div>
-        </NuxtLink>
-        <div class="hidden sm:flex grow" />
-        <div class="flex items-center">
-          <ProjectPageModelsCardUpdatedTime
-            :updated-at="updatedAtFullDate"
-            :class="`text-xs w-full text-foreground-2 sm:mr-1 truncate transition ${
-              hovered ? 'sm:w-auto' : 'sm:w-0'
-            }`"
-          />
           <FormButton
-            v-if="finalShowVersions"
-            v-tippy="'View Version Gallery'"
-            rounded
-            size="xs"
-            :to="modelVersionsRoute(projectId, model.id)"
-            :class="`transition gap-0.5 ml-1 ${
-              hovered ? 'inline-block opacity-100' : 'sm:hidden sm:opacity-0'
-            }`"
+            v-tippy="'View Versions'"
+            color="subtle"
+            size="sm"
+            class="flex items-center gap-1 !text-foreground-2"
+            @click.stop="router.push(modelVersionsRoute(projectId, model.id))"
```

Comment:
It was decided by design that the whole model card is now clickable, rather than just the title and the preview image. 

There is a click event on the outer div. Without using the click.stop, the version button didn't work, instead you always navigated to the model page, not the model version page. 

Happy to revisit this if you think there is a better method for handling nested links.